components/function-controller/internal/webhook/conversion_webhook.go

```diff
@@ -47,17 +47,21 @@ func NewConvertingWebhook(client ctrlclient.Client, scheme *runtime.Scheme, log
 }
 
 func (w *ConvertingWebhook) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
+	log := w.log
 	convertReview := &apix.ConversionReview{}
 	err := json.NewDecoder(req.Body).Decode(convertReview)
 	if err != nil {
-		w.log.Error(err, "failed to read conversion request")
+		log.Error(err, "failed to read conversion request")
 		resp.WriteHeader(http.StatusBadRequest)
 		return
 	}
 
+	log = w.log.With("uid", convertReview.Request.UID)
```

Comment:
Do we need to know the uid of request?
Can we get the name of converted resource?

Reply:
I set up the logger with UID because this is the only info from the request struct that informs us from which request this log is (to help with cases when we would have a few requests at the same time). What do you think?